packages/tap-runner/test/unit/tap-helper.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+
+import { parseArguments } from '../../src/tap-helper.js';
+
+describe('tap-helper', () => {
```

Comment:
Great to see some more unit tests 🤩.

Maybe also add tests for multiple instances of the placeholder.